tests/test-pe.c

```diff
@@ -587,6 +587,16 @@ int main(int argc, char** argv)
       "tests/data/ca21e1c32065352d352be6cde97f89c141d7737ea92434831f998080783d5386");
 
 
+  assert_true_rule_file(
```

Comment:
A test case with a file that actually has a section like this is important for making sure that this feature is working as intended.

Reply:
Added `pe_mingw` (program that print `Hello`) which was compiled by mingw into `tests/data`. This is then used in added test to test `full_name`